test/local/account_routes.js

```diff
@@ -56,699 +56,461 @@ var makeRoutes = function (options) {
-          verifierSetAt: 0,
-          email: TEST_EMAIL
+      return runTest(route, mockRequest, function (response) {
+        t.deepEqual(response, {
+          email: TEST_EMAIL,
+          verified: false,
+          sessionVerified: false,
+          emailVerified: true
         })
       })
-    }
-    var mockCustoms = {
-      reset: sinon.spy(function (email) {
-        return P.resolve()
-      })
-    }
-    var mockPush = {
-      notifyUpdate: sinon.spy(function () {})
-    }
-    var accountRoutes = makeRoutes({
-      db: mockDB,
-      customs: mockCustoms,
-      push: mockPush
-    })
-
-    return new P(function(resolve) {
-      getRoute(accountRoutes, '/account/reset')
-        .handler(mockRequest, function(response) {
-          resolve(response)
-        })
-    })
-    .then(function(response) {
-      t.equal(mockDB.resetAccount.callCount, 1)
-
-      t.equal(mockPush.notifyUpdate.callCount, 1)
-      t.equal(mockPush.notifyUpdate.firstCall.args[0], uid.toString('hex'))
-      t.equal(mockPush.notifyUpdate.firstCall.args[1], 'passwordReset')
-
-      t.equal(mockDB.account.callCount, 1)
-      t.equal(mockCustoms.reset.callCount, 1)
+    }, t)
+  }, t)
+})
+
+test('/account/reset', function (t) {
+  var uid = uuid.v4('binary')
+  var mockRequest = mocks.mockRequest({
+    credentials: {
+      uid: uid.toString('hex')
+    },
+    payload: {
+      authPW: crypto.randomBytes(32).toString('hex')
+    }
+  })
+  var mockDB = mocks.mockDB({
+    uid: uid,
+    email: TEST_EMAIL
+  })
+  var mockCustoms = {
+    reset: sinon.spy(function () {
+      return P.resolve()
     })
   }
-)
-
-test(
-  'device updates dont write to the db if nothing has changed',
-  function (t) {
-    var uid = uuid.v4('binary')
-    var deviceId = crypto.randomBytes(16)
-    var mockRequest = {
-      auth: {
-        credentials: {
-          uid: uid.toString('hex'),
-          deviceId: deviceId,
-          deviceName: 'my awesome device',
-          deviceType: 'desktop',
-          deviceCallbackURL: '',
-          deviceCallbackPublicKey: '',
-        }
-      },
-      payload: {
-        id: deviceId.toString('hex'),
-        name: 'my awesome device'
-      }
-    }
-    var mockDB = {
-      updateDevice: sinon.spy(function () {
-        return P.resolve()
-      })
-    }
-    var mockLog = mocks.spyLog()
-    var accountRoutes = makeRoutes({
-      db: mockDB,
-      log: mockLog
-    })
-    return new P(function(resolve) {
-      getRoute(accountRoutes, '/account/device')
-        .handler(mockRequest, function(response) {
-          resolve(response)
-        })
-    })
-    .then(function(response) {
-      t.equal(mockDB.updateDevice.callCount, 0, 'updateDevice was not called')
-
-      t.equal(mockLog.increment.callCount, 1, 'a counter was incremented')
-      t.equal(mockLog.increment.firstCall.args[0], 'device.update.spurious')
-
-      t.deepEqual(response, mockRequest.payload)
-    })
+  var mockPush = {
```

Comment:
Wonder if it is worthwhile to pull mockPush into the mocks file? Seems to be use a few times in this file.